### PR TITLE
8244976: vmTestbase/nsk/jdi/Event/request/request001.java doesn' initialize eName

### PR DESCRIPTION
**Pull request by lmesnik**
Fixed tests that use eName to print event name to print the class name. Plus some minor refactoring.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8244976](https://bugs.openjdk.org/browse/JDK-8244976): vmTestbase/nsk/jdi/Event/request/request001.java doesn' initialize eName

### Reviewers
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**) ⚠️ Review applies to [c6f79264](https://git.openjdk.org/jdk/pull/9624/files/c6f79264357807ffeab18b0ae7932b8b69cd4adb)
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9624/head:pull/9624` \
`$ git checkout pull/9624`

Update a local copy of the PR: \
`$ git checkout pull/9624` \
`$ git pull https://git.openjdk.org/jdk pull/9624/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9624`

View PR using the GUI difftool: \
`$ git pr show -t 9624`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9624.diff">https://git.openjdk.org/jdk/pull/9624.diff</a>

</details>

### CODE REVIEW

**Review by shipilev — APPROVED** [review]
Looks reasonable.

File: test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java [hunk 1]
```diff
@@ -554,11 +550,14 @@ private void testRun()
                 } else {
                     log3("ERROR: else clause in detecting type of event1");
                     testExitCode = FAILED;
+                    throw new JDITestRuntimeException("** unexpected event **");
```
Comment (sspitsyn) [inline test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java:553]: It is better to print the event type (or, at least, something) in this case.

File: test/hotspot/jtreg/vmTestbase/nsk/jdi/EventIterator/nextEvent/nextevent001.java [hunk 2]
```diff
@@ -543,11 +540,14 @@ private void testRun()
                 } else {
                     log3("ERROR: else clause in detecting type of event1");
                     testExitCode = FAILED;
+                    throw new JDITestRuntimeException("** unexpected event **");
```
Comment (sspitsyn) [inline test/hotspot/jtreg/vmTestbase/nsk/jdi/EventIterator/nextEvent/nextevent001.java:543]: The same as commented above.

**Review by sspitsyn — APPROVED** [review]
I've added a couple of nits.
Otherwise, it looks good.
Thanks,
Serguei

File: test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java (hunk 1 above)
Comment (plummercj) [reply test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java:553]: Just add ` + event`.

File: test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java [hunk 3]
```diff
@@ -554,11 +550,14 @@ private void testRun()
...
                 }
+                log2("--------> got: " + event1.getClass().getName() + " index: " + index);
 
-                flag = 1 << index;
+                int flag = 1 << index;
                 if ((flagsCopy & flag) == 0) {
-                    log3("ERROR: event duplication: " + eName);
+                    log3("ERROR: event duplication. flafsCopy = " + Integer.toBinaryString(flagsCopy)
+                            + " flag = " + Integer.toBinaryString(flag));
```
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java:560]: typo: flafsCopy

I think you should print the event also.

File: test/hotspot/jtreg/vmTestbase/nsk/jdi/LocatableEvent/thread/thread001.java [hunk 4]
```diff
@@ -417,45 +417,35 @@ private void testRun()
...
                 } else if (event1 instanceof StepEvent ) {
-                    eName = "StepEvent";
                     index = 6;
                 } else {
                     log3("ERROR: else clause in detecting type of event1");
                     testExitCode = FAILED;
                     throw new JDITestRuntimeException("** unexpected event **");
                 }
-                log2("--------> got: " + eName);
+                log2("--------> got: " + event1.getClass().getName() + " index: " + index);
```
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/jdi/LocatableEvent/thread/thread001.java:448]: Why not print the event rather than just its name.

**Review by plummercj — CHANGES_REQUESTED** [review]

**Review by sspitsyn — APPROVED** [review]

**Review by plummercj — APPROVED** [review]

**Pull request closed without merging**